yt_dlp/extractor/tumblr.py

```diff
@@ -14,31 +15,129 @@ class TumblrIE(InfoExtractor):
     _VALID_URL = r'https?://(?P<blog_name>[^/?#&]+)\.tumblr\.com/(?:post|video)/(?P<id>[0-9]+)(?:$|[/?#])'
     _NETRC_MACHINE = 'tumblr'
     _LOGIN_URL = 'https://www.tumblr.com/login'
+    _OAUTH_URL = 'https://www.tumblr.com/api/v2/oauth2/token'
     _TESTS = [{
         'url': 'http://tatianamaslanydaily.tumblr.com/post/54196191430/orphan-black-dvd-extra-behind-the-scenes',
         'md5': '479bb068e5b16462f5176a6828829767',
         'info_dict': {
             'id': '54196191430',
             'ext': 'mp4',
-            'title': 'tatiana maslany news, Orphan Black || DVD extra - behind the scenes ↳...',
+            'title': 'md5:dfac39636969fe6bf1caa2d50405f069',
             'description': 'md5:390ab77358960235b6937ab3b8528956',
+            'uploader_id': 'tatianamaslanydaily',
+            'uploader_url': 'https://tatianamaslanydaily.tumblr.com/',
             'thumbnail': r're:^https?://.*\.jpg',
             'duration': 127,
+            'like_count': int,
+            'repost_count': int,
+            'age_limit': 0,
+            'tags': ['Orphan Black', 'Tatiana Maslany', 'Interview', 'Video', 'OB S1 DVD Extras'],
         }
     }, {
+        'note': 'multiple formats',
         'url': 'https://maskofthedragon.tumblr.com/post/626907179849564160/mona-talking-in-english',
         'md5': 'f43ff8a8861712b6cf0e0c2bd84cfc68',
         'info_dict': {
             'id': '626907179849564160',
             'ext': 'mp4',
-            'title': 'Me roast is buggered!, Mona\xa0“talking” in\xa0“english”',
+            'title': 'Mona\xa0“talking” in\xa0“english”',
             'description': 'md5:082a3a621530cb786ad2b7592a6d9e2c',
+            'uploader_id': 'maskofthedragon',
+            'uploader_url': 'https://maskofthedragon.tumblr.com/',
             'thumbnail': r're:^https?://.*\.jpg',
             'duration': 7,
+            'like_count': int,
+            'repost_count': int,
+            'age_limit': 0,
+            'tags': 'count:19',
         },
         'params': {
             'format': 'hd',
         },
+    }, {
+        'note': 'non-iframe video (with related posts)',
+        'url': 'https://shieldfoss.tumblr.com/post/675519763813908480',
+        'md5': '12bdb75661ef443bffe5a4dac1dbf118',
+        'info_dict': {
+            'id': '675519763813908480',
+            'ext': 'mp4',
+            'title': 'Shieldfoss',
+            'uploader_id': 'nerviovago',
+            'uploader_url': 'https://nerviovago.tumblr.com/',
+            'thumbnail': r're:^https?://.*\.jpg',
+            'like_count': int,
+            'repost_count': int,
+            'age_limit': 0,
+            'tags': [],
+        }
+    }, {
+        'note': 'dashboard only (original post)',
```

Comment:
i didn't know `note` can be used for this...
I always thought we have to stick to comments...

Reply:
I'm here aping the usage in some of the YouTube tests. (I don't know whether `note` has any practical significance.)

Reply:
Many extractors use note, many use comments 🤷‍♂️.There isn't really any practical difference b/w them